docs/docs/tutorial/debugger.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 11
+---
+
+# Debugger    
+
+Debugger is on the left-sidebar of the app builder. In case of a query fails, the error logs can be viewed in a popover when
```

Comment:
` In case of a query fails, the error logs can be viewed in a popover when` ->
The debugger captures errors in queries. For example, when a database query fails due to the unavailability of a database or when a REST API query fails due to an incorrect URL, the errors will be displayed on the debugger. The debugger also displays relevant data related to the error along with the error message.